zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/MappingDeleteProcessor.java

```diff
@@ -140,12 +140,17 @@ private void deleteMapping(final PersistedMapping mapping) {
               .setEntityId(mapping.getMappingId())
               .setEntityType(EntityType.MAPPING));
     }
-    for (final var roleKey : mapping.getRoleKeysList()) {
+    for (final var roleId :
```

Comment:
❓ I think this change to `roleId` may be a little confusing as this will return a list of `roleKeys` (this is what I _think_ is happening based on the comments mentioning using roleId instead of key. Its NABD as it'lldisappear soon but I can imagine someone reading this and assuming that we do already use a roleId and just removing the TODOs.